weblate_web/models.py

```diff
@@ -242,7 +242,7 @@ def send_notification(self, notification):
 
 def process_donation(payment):
     if payment.state != Payment.ACCEPTED:
-        raise ValueError("Can not process not accepted payment")
+        raise ValueError("Can not process non-accepted payment")
```

Comment:
Not 100% here. What happens?

Reply:
Payment was accepted by the payment gateway - was not paid. User should not get here at all, this is just a safeguard in case somebody tries to cheat.

Reply:
OK. Think it works then.